### PR TITLE
Adds petty officer uniform and preliminary loadout

### PR DESCRIPTION
**Pull request by Vivalas2**
Takes an old custom sprite and makes it into the petty officer's uniform, and adds the parade hat as his hat. (Courtesy of Finalsong) Gives Petty Officer grey gloves, a security PDA and headset, and black shoes.

### CODE REVIEW

File: code/game/jobs/job/government_sector.dm [hunk 1]
```diff
@@ -7,4 +7,16 @@
 	spawn_positions = 4
 	supervisors = "Jovian naval officials and Sol colonial law"
 	access = list()
-	alt_titles = list("Sub Pilot", "Peace Officer")
\ No newline at end of file
+	alt_titles = list("Sub Pilot", "Peace Officer")
+
+
+	equip(var/mob/living/carbon/human/H)
```
Comment (Zuhayr) [inline code/game/jobs/job/government_sector.dm]: Absolute paths please!

**Comment by Zuhayr** [comment]
Other than the absolute path thing, :+1:

File: code/game/jobs/job/government_sector.dm (hunk 1 above)
Comment (Vivalas2) [inline code/game/jobs/job/government_sector.dm]: Hm? I kinda copied that from the Hos loadout

Comment (Vivalas2) [inline code/game/jobs/job/government_sector.dm]: @Zuhayr what do you mean?

Comment (GinjaNinja32) [inline code/game/jobs/job/government_sector.dm]: What you currently have:
```DM
/datum/job/government
	// things
	equip(var/mob/living/carbon/human/H)
		if(!H)	return 0
		H.equip_to_slot_or_del(new /obj/item/device/radio/headset/headset_sec(H), slot_l_ear)
		H.equip_to_slot_or_del(new /obj/item/clothing/under/rank/petty_officer(H), slot_w_uniform)
		// more things
```
What Zuhayr (and I, and probably everyone else) would like:
```DM
/datum/job/government
	// things
/datum/job/government/equip(var/mob/living/carbon/human/H)
	if(!H)	return 0
	H.equip_to_slot_or_del(new /obj/item/device/radio/headset/headset_sec(H), slot_l_ear)
	H.equip_to_slot_or_del(new /obj/item/clothing/under/rank/petty_officer(H), slot_w_uniform)
	// more things
```

Comment (Vivalas) [inline code/game/jobs/job/government_sector.dm]: Oh, yeah, sure. I didn't catch that.

**Comment by Nienhaus** [comment]
Remember to upload screenshots guys.

**Comment by Vivalas2** [comment]
image

File: code/game/jobs/job/government_sector.dm [hunk 2]
```diff
@@ -7,4 +7,16 @@
 	spawn_positions = 4
 	supervisors = "Jovian naval officials and Sol colonial law"
 	access = list()
-	alt_titles = list("Sub Pilot", "Peace Officer")
\ No newline at end of file
+	alt_titles = list("Sub Pilot", "Peace Officer")
+
+
+	/datum/job/government/equip(var/mob/living/carbon/human/H)
```
Comment (Zuhayr) [inline code/game/jobs/job/government_sector.dm]: Does this compile? It shouldn't have a tab before the path.

Comment (Vivalas2) [inline code/game/jobs/job/government_sector.dm]: No, It probably won't, changed it in the online-editor

**Pull request merged**